### PR TITLE
Add details about how to build

### PR DESCRIPTION
**Pull request by XihuanYang**
Signed-off-by: Xihuan Yang <xihuan.yang@citrix.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://travis-ci.org/xenserver/xe-guest-utilities.svg?branch=master)](https://travis-ci.org/xenserver/xe-guest-utilities)
 
-go-guest-utilites
+Introduction
 ===================
 
 This is the golang guest utilities for XenServer
```
Comment (minli1) [inline README.md:6]: This is the golang guest utilities for XenServer -> This is the golang guest utilities for Citrix Hypervisor

**Review by minli1 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -21,3 +21,57 @@ Build Instructions
 [Go development environment](https://golang.org/doc/install) is required to build the guest utilities.
 
 Type `make` or `make build` to build the xenstore and xe-daemon.
+
+USE $GOPATH to build
+-----------
+e.g. use go1.6.4 to build
```
Comment (liulinC) [inline README.md:27]: exactly 1.6.4? can higher version build?

**Review by xihuan-citrix — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -21,3 +21,57 @@ Build Instructions
...
+`make`
+
+Enable $GO111MODULE to build
+-----------
+e.g. use go1.6.4 to build
+1. Download xe-guest-utilities
+
+`cd /root`
+
+`git clone git@github.com:xenserver/xe-guest-utilities.git`
```
Comment (liulinC) [inline README.md:62]: why clone xe-guest-utilities.git twice ?

File: README.md [hunk 4]
```diff
@@ -21,3 +21,57 @@ Build Instructions
...
+2. Enable GO111MODULE
+
+`export GO111MODULE=on`
```
Comment (liulinC) [inline README.md:65]: Looks requires `GO111module=on` when outside GOPATH and `off` inside, 
try `GO111MODULE=auto` ?   https://maelvls.dev/go111module-everywhere/

**Review by liulinC — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (xihuan-citrix) [reply README.md:6]: Raise another PR to fix all these including other citrix related name

File: README.md [hunk 5]
```diff
@@ -21,3 +21,57 @@ Build Instructions
...
+
+3. Initializing a project
+
+`go mod init xe-guest-utilities`
+
+`go mod tidy`
+
```
Comment (4censord) [inline README.md:72]: This step isn't required since #104 has been merged

File: README.md (hunk 4 above)
Comment (4censord) [reply README.md:65]: Since go 1.16 this is enabled by default

File: README.md [hunk 6]
```diff
@@ -21,3 +21,57 @@ Build Instructions
...
+`export GO111MODULE=off`
+
+4. Download golang.org/x/sys
+
+`mkdir -p $GOPATH/src/golang.org/x`
+
+`cd $GOPATH/src/golang.org/x`
+
+`git clone http://github.com/golang/sys.git`
+
```
Comment (4censord) [inline README.md:48]: go has a shorthand for this:
`go get golang.org/x/sys`

**Pull request merged**